pkgs/build-support/dotnet/make-nuget-deps/default.nix

```diff
@@ -1,14 +1,30 @@
-{ linkFarmFromDrvs, fetchurl }:
+{ linkFarmFromDrvs, substituteAll, fetchurl, stdenvNoCC, zip }:
```

Comment:
I think `substituteAll` is unused now

Reply:
Huh, I wonder why my editor didn't tell me that!